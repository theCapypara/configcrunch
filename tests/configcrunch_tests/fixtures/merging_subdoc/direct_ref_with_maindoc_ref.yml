base:
    $ref: /base_merge
    str_field: test
    level_direct:
        $ref: /level1
        more:
          merge_test: true
