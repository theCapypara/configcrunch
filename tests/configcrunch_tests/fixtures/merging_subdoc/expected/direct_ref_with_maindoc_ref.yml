base:
    str_field: test
    level_dict:
        base_merge:
            $name: base_merge
            name: level2
            more:
                from: base_merge
        from_base_merge:
            $name: from_base_merge
            name: no-merge
    level_direct:
        name: level1
        more:
          merge_test: true
          from: base_merge